### PR TITLE
Add Requestrr Subdomain and Subfolder Config

### PR DESCRIPTION
**Pull request by kiityman1**
https://github.com/darkalfx/requestrr

[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

File: requestrr.subfolder.conf.sample [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+# edit /config/configs/znc.conf and add URIPrefix = /znc/ in the line above </Listener> and restart the znc container
```
Comment (nemchik) [inline requestrr.subfolder.conf.sample:2]: Can you update this comment with instructions specific to requestrr? Does it need a setting for a base url?

Also please remove the blank line above it.

Comment (kiityman1) [reply requestrr.subfolder.conf.sample:2]: Could you only merge the subdomain one? I don't use subfolders aand don't have it installed atm

Comment (nemchik) [reply requestrr.subfolder.conf.sample:2]: You would have to remove the subfolder proxy from your request.

Comment (kiityman1) [reply requestrr.subfolder.conf.sample]: Done

**Pull request merged**